docs/content/deprecation/features.md

```diff
@@ -26,3 +28,11 @@ please use the `namespaces` options instead.
 ### TLS 1.0 and 1.1
 
 Starting on 2.8 the default TLS options will use the minimum version of TLS 1.2. Of course, it can still be overridden with custom configuration.  
+
+### Kubernetes CRDs API Group "traefik.containo.us"
+
+In v2.10, the Kubernetes CRDs API Group "traefik.containo.us" is deprecated, and its support will end starting with Traefik v3. Please use the API Group "traefik.io" instead.
+
+### Kubernetes CRDs API Version "traefik.io/v1alpha1"
+
+In v2.10, the Kubernetes CRDs API Version "traefik.io/v1alpha1" will not be supported in Traefik v3 itself.
```

Comment:
```suggestion
Starting with v3, the Kubernetes CRDs API Version "traefik.io/v1alpha1" is not supported.
```

Reply:
We could mention `traefik v3` instead of just `v3`, and we could use backticks(``` ` ```) to put an emphasis on the CRD version (e.g., ``` `containo.us/v1alpha1` ``` instead of ``` "containo.us/v1alpha1" ```). WDYT ?

Reply:
```suggestion
The newly introduced Kubernetes CRD API Version `traefik.io/v1alpha1` will subsequently be removed in Traefik v3. The following version will be `traefik.io/v1`.
```